tests/integration/test_backend.py

```diff
@@ -24,7 +24,7 @@
 
 
 def test_cleanup_containers():
-    with DockerBackend(logging_level=logging.DEBUG) as backend:
+    with DockerBackend(logging_level=logging.DEBUG, cleanup=True) as backend:
```

Comment:
I think default value should be cleanup=False - since now conu is presented as tool not only for testing. But I'd like to see what, you think @rpitonak, @TomasTomecek?

Reply:
absolutely agree, cleanup should equal to false by default